apps/ta_dump.py

```diff
@@ -10,69 +10,83 @@
 import numpy as np
 import matplotlib.pyplot as plt
 from matplotlib.backends.backend_pdf import PdfPages
+from scipy import stats
 
 import trgtools
 
-TICK_TO_SEC_SCALE = 512e-9 # s per tick
+TICK_TO_SEC_SCALE = 16e-9 # s per tick
 
-def window_length_hist(window_lengths):
+def window_length_hist(window_lengths, seconds=False):
```

Comment:
In the further future, I can imagine there being some form of TPPlots class that holds a data member seconds, that can be accessed from any plotting function so it doesn't need to be passed etc.

Doesn't matter for now, just throwing ideas for the further future if we re-design this.